ros2topic/test/test_echo_pub.py

```diff
@@ -176,6 +176,32 @@ def message_callback(msg):
                 finally:
                     # Cleanup
                     self.node.destroy_subscription(subscription)
+    
+    @launch_testing.markers.retry_on_failure(times=5)
+    def test_pub_basic(self, launch_service, proc_info, proc_output):
+        command_action = ExecuteProcess(
+            cmd=(['ros2', 'topic', 'pub', '-t', '5', '--max-wait-time', '1', '/clitest/topic/pub_times',
+                  'std_msgs/String', 'data: hello']),
+            additional_env={
+                'PYTHONUNBUFFERED': '1'
+            },
+            output='screen'
+        )
+        with launch_testing.tools.launch_process(
+            launch_service, command_action, proc_info, proc_output,
+            output_filter=launch_testing_ros.tools.basic_output_filter(
+                filtered_rmw_implementation=get_rmw_implementation_identifier()
+            )
+        ) as command:
+            assert command.wait_for_shutdown(timeout=2)
```

Comment:
Let's increase this timeout to 5, again to make sure Windows has time to actually launch the process.

Reply:
See c6dda6e